MdeModulePkg/Universal/SetupBrowserDxe/IfrParse.c

```diff
@@ -1821,23 +1821,35 @@ ParseOpCodes (
       case EFI_IFR_SUBTITLE_OP:
         CurrentStatement = CreateStatement (OpCodeData, FormSet, CurrentForm);
         ASSERT (CurrentStatement != NULL);
+        // MU_CHANGE - Verify valid before pointer dereference
+        if (CurrentStatement != NULL) {

```

Comment:
Return error codes for the failed cases?

Reply:
Return codes are not necessary. The CreateStatement would allocate memory, and then it would add it the allocated memory to the current form's IFR statements.
https://github.com/microsoft/mu_basecore/blob/release/202208/MdeModulePkg/Universal/SetupBrowserDxe/IfrParse.c#L82

A failure to create the statement doesn't prevent the code from being able to continue, because the statement will just not exist in the form.